### PR TITLE
ISPN-11071 Support container server driver image troubleshooting

### PR DESCRIPTION
**Pull request by diegolovison**
https://issues.redhat.com/browse/ISPN-11071

### CODE REVIEW

**Comment by diegolovison** [comment]
My PR doesn't fix the issue. It helps to troubleshooting only.

File: server/runtime/src/test/java/org/infinispan/server/test/ContainerInfinispanServerDriver.java [hunk 1]
```diff
@@ -56,7 +58,9 @@
    CountdownLatchLoggingConsumer latch;
    ImageFromDockerfile image;
    private File rootDir;
+
    private final boolean preferContainerExposedPorts = Boolean.getBoolean("org.infinispan.test.server.container.preferContainerExposedPorts");
+   private final boolean deleteImageOnExit = Boolean.getBoolean("org.infinispan.test.server.container.deleteImageOnExit");
```
Comment (tristantarrant) [inline server/runtime/src/test/java/org/infinispan/server/test/ContainerInfinispanServerDriver.java]: This should be true by default

Comment (diegolovison) [reply server/runtime/src/test/java/org/infinispan/server/test/ContainerInfinispanServerDriver.java]: In this case, do we need the argument `org.infinispan.test.server.container.deleteImageOnExit` ?

Comment (tristantarrant) [reply server/runtime/src/test/java/org/infinispan/server/test/ContainerInfinispanServerDriver.java]: Yeah, I don't think you're ever going to look at the image after a test execution

Comment (diegolovison) [reply server/runtime/src/test/java/org/infinispan/server/test/ContainerInfinispanServerDriver.java]: Done

**Comment by ryanemerson** [comment]
Thanks @diegolovison

**Comment by tristantarrant** [comment]
Unfortunately the code was not ready for integration. I have fixed the delete on exit in my #7843

**Comment by tristantarrant** [comment]
I will also include a proper backported fix for this in #7844

**Pull request merged**